### PR TITLE
feat(chart-controls): move cross-filtering control to data panel

### PR DESCRIPTION
**Pull request by stephenLYZ**
🏆 Enhancements
- Move the cross filter control from the "Customize" tab to the "Query" tab under the "(adhoc) filter" control.
- Rename from "Enable emitting filters" to "Emit dashboard cross filters"

releated: https://github.com/apache/superset/issues/15910

### CODE REVIEW

File: plugins/plugin-chart-echarts/src/BoxPlot/controlPanel.ts [hunk 1]
```diff
@@ -37,6 +35,7 @@ export default {
       controlSetRows: [
         ['metrics'],
         ['adhoc_filters'],
+        emitFilterControl,
```
Comment (zhaoyongjie) [inline plugins/plugin-chart-echarts/src/BoxPlot/controlPanel.ts:38]: We should use `sections.emitFilterControl` for consistency namespace.

File: plugins/plugin-chart-echarts/src/Funnel/controlPanel.tsx [hunk 2]
```diff
@@ -43,6 +44,7 @@ const config: ControlPanelConfig = {
         ['groupby'],
         ['metric'],
         ['adhoc_filters'],
+        emitFilterControl,
```
Comment (zhaoyongjie) [inline plugins/plugin-chart-echarts/src/Funnel/controlPanel.tsx:47]: same before.

File: plugins/plugin-chart-echarts/src/BoxPlot/controlPanel.ts (hunk 1 above)
Comment (villebro) [reply plugins/plugin-chart-echarts/src/BoxPlot/controlPanel.ts:38]: I think this is ok - IMO `sections`  should be reserved for `ControlPanelSectionConfig` objects, not `ControlSetRow` objects.

**Review by villebro — APPROVED** [review]
LGTM, but it seems you need to add a unit test for when the feature flag is set and unset (codecov needs to be 100 % for the modules under `packages/`).

**Pull request merged**